### ISSUE TITLE
What are the conditions under which a single entity is included multiple times in a response?

### ISSUE THREAD

**Issue by oh344225** [microsoftgraph/microsoft-graph-docs#16657]
What are the conditions under which a single entity is included multiple times in a response?

I want to know the specific conditions.
Please tell me the specific conditions.
```
Note: It's possible that a single entity will be contained multiple times in the response, if that entity was changed multiple times and under certain conditions. Delta queries enable your application to list all the changes, but can't ensure that entities are unified in a single response.
```

Looking forward to receiving the revised document.

---
#### ドキュメントの詳細

⚠ *このセクションを編集しないでください。 docs.microsoft.com で必須です ➟ GitHub の問題のリンク。*

* ID: 66201f1c-504b-eba2-f489-c611523fdf8d
* Version Independent ID: 5c711b4f-e234-ae03-53e5-ecd206a157ff
* Content: [デルタ クエリを使用して、Microsoft Graph データの変更を追跡する - Microsoft Graph](https://docs.microsoft.com/ja-jp/graph/delta-query-overview#resource-representation-in-the-delta-query-response)
* Content Source: [concepts/delta-query-overview.md](https://github.com/microsoftgraph/microsoft-graph-docs.ja-JP/blob/main/concepts/delta-query-overview.md)
* Product: **non-product-specific**
* Technology: **microsoft-graph**
* GitHub Login: @FaithOmbongi
* Microsoft Alias: **MSGraphDocsVteam**

**Comment by FaithOmbongi** [comment]
Hi @oh344225, let us get back to you on this.

cc @Jumaodhiss

**Comment by oh344225** [comment]
Hi,@Jumaodhiss, How is the current status?

I want to know about Resource representation in the delta query response. 
It says "Note: It's possible that a single entity will be contained multiple times in the response, if that entity was changed multiple times and under certain conditions. Delta queries enable your application to list all the changes, but can't ensure that entities are unified in a single response. "

In what cases does this happen?
I want to know the specific conditions.

I am looking forward to your response.

**Comment by FaithOmbongi** [comment]
Hi @oh344225 -

The underlying database that Microsoft Graph queries is highly distributed and sometimes there may be replication delays of a few seconds up to minutes. This also applies to change tracking and is one condition when a single entity, updated multiple times, may appear multiple times in the delta responses.

For example, I changed the **displayName** and **members** for a group. When I queried the /delta less than a 30seconds after the changes, I received two separate delta responses - one for the change to displayName and the other for the change to members. Another time I added members and owners and waited for 5 minutes, I received only one delta response for both changes.

I hope this answers your question.
